### DISCUSSION TITLE
VLC Open Map Stream - Check admin password

### DISCUSSION THREAD

**Discussion by Slingboxforever** [GerryDazoo/Slinger#470] [General]
Hi All,

Looking for some help on the config admin password error seen below.  thank you

Version : 3.08f Running on Windows-10-10.0.22621-SP0 pid= 11020 C:\Users\jpepp\OneDrive\Documents\slinger\Slinger-main\slingbox_server.exe
Using config file config.ini
Connection Manager Running on port 8080 with 10 max streams using URL slingbox.
Streamer Running:  10 config.ini SLINGBOX 8080 8080 8388608
starting up on  port 8080 8388608
 * Serving Flask app 'slingbox_server' (lazy loading)
 * Environment: production
   WARNING: This is a development server. Do not use it in a production deployment.
   Use a production WSGI server instead.
 * Debug mode: off
8080 No valid slingbox ip info found in config.ini
Finding Slingbox on local network. My IP Info =  192.168.1.11
Slingbox Found 192.168.1.12 5204 " My Slingbox " FinderID 238EF85F3619AB928409E9737F9BC42B
8080 Using slingbox at  ('192.168.1.12', 5204)
Streamer:  8080 Waiting for first stream, flushing any IR requests that arrive while not connected to slingbox
12/31/2022, 17:57:48.081   Streaming request from ('192.168.1.11', 53313)

Slinginfo  "350/500/M1/M2" 12 30 192.168.1.12 5204 3072 10
8080 Starting Stream for  192.168.1.11:53313
Connecting... ('192.168.1.12', 5204) Control
cmd: 0x67 err: 0x2 8
8080 Error Starting Session. Check your admin password in config.ini file!
8080 ERROR: Slingbox session startup failed.
8080 Logging Out
Streamer:  8080 Waiting for first stream, flushing any IR requests that arrive while not connected to slingbox

Config file setup below:

[SLINGBOX]
sbtype="350/500/M1/M2"
;sbtype="Solo/Pro/ProHD"
;sbtype="240"
password="akKJpB1Scw84CqL"

;uncomment the next lines ipaddress and port with your local network info
;if you don't want the server to automatically find your 
;slingbox on the local network. Needed if server and slingbox are
;not on the same LAN segment. Also removes requirement for the netifaces module  
;"ipaddress=192.168.1.150"
;"port=5212"

;Replace with your finderid, If and only if you need remote access and don't  
;have a static ip and you plan on using my service to access you server remotely. 
;Please read the release notes. Most people don't need this.
;finderid=238EF85F3619AB928409E9737F9BC42B

**Comment by lepmail** [comment]
You need to remove the quotes around the password on the config. Happy New Year!

**Reply by Slingboxforever** [reply]
Yep.  that's it.  thank you!!!  happy New Year!
